tasks/extract_filepaths_for_granule/README.md

```diff
@@ -80,6 +80,7 @@ resource "aws_lambda_function" "extract_filepaths_for_granule" {
 The lambda handler event excepts a dictionary having a list of granules as input. Check the input schema [here](https://github.com/nasa/cumulus-orca/blob/master/tasks/extract_filepaths_for_granule/schemas/input.json) and the configuration schema [here](https://github.com/nasa/cumulus-orca/blob/master/tasks/extract_filepaths_for_granule/schemas/config.json). An example input to the Lambda function can be seen below.
 ```
 {
+      "payload":{
```

Comment:
Missing indent for this section.

Reply:
For clarification, this line needs more or less indentation? I it needs one less tab but wanted to check

Reply:
I was just noting that `payload` and `granules` were at the same indentation. Yeah, just move everything one tab to the left and it should be fine.

Reply:
task_config is over-indented as well.

Reply:
Understood, just pushed the fixes to removed the excess of indentations